src/IRutils.cpp

```diff
@@ -112,6 +112,7 @@ std::string typeToString(const decode_type_t protocol,
     case GREE:          result = "GREE";              break;
     case HAIER_AC:      result = "HAIER_AC";          break;
     case HITACHI_AC:    result = "HITACHI_AC";        break;
+    case HITACHI_AC1:   result = "HITACHI_AC1";       break;
```

Comment:
Should we also have a `case HITACHI_AC2:   result = "HITACHI_AC2";` here?

Reply:
Totally. Excellent catch. Thanks.

Fixed plus found a few other subtle trivial omissions. e.g. code that should be included via #defines etc.

PTAL